src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AzureFeature.kt

```diff
@@ -36,15 +37,26 @@ class AzureFeature(
     private val auroraApim = AuroraAzureApimSubPart()
 
     override fun isActive(spec: AuroraDeploymentSpec): Boolean {
-        return spec.isJwtToStsConverterEnabled || spec.isAzureAppFqdnEnabled || spec.isApimEnabled
+        return !isAzureSpecificallyDisabled(spec) &&
+            (spec.isJwtToStsConverterEnabled || spec.isAzureAppFqdnEnabled || spec.isApimEnabled)
     }
 
+    private fun isAzureSpecificallyDisabled(spec: AuroraDeploymentSpec): Boolean =
+        spec.getOrNull<String>("azure")?.let { it == "false" } == true
```

Comment:
hva med:
```
private fun isAzureSpecificallyDisabled(spec: AuroraDeploymentSpec): Boolean = spec.isSimplifiedAndDisabled("azure")
```

Reply:
nei, hvis azure ikke er satt (ingen azure-tagg) feiler denne med nullpointerexception fordi isSimplifiedAndDisabled() kaller spec.get<Boolean>() i stedet for spec.getOrNull<Boolean>().

Reply:
Det hadde vel blitt løst hvis man sette defaultValue = false for AuroraConfigFieldHandler for azure simplified handleren.

Reply:
Uansett, så synes jeg det er en litt rar konstruksjon slik det står nå. 
En mulig forbedring hvis du ikke tar opp forslaget med defaultvalue:
```
    private fun isAzureSpecificallyDisabled(spec: AuroraDeploymentSpec): Boolean =
        spec.getOrNull<Boolean>("azure") == true
```

Reply:
defaultValue fungerer dårlig; dersom defaultValue er "true" vil mange reagere på å ha "slått det på" i `ao get spec` selv om de ikke bruker azure. Dersom defaultValue er "false" vil vi ikke kunne gjenkjenne at det er slått av. Så defaultValue=null er greit, for det blir ikke listet ut i `ao get spec`. 
Men jeg er enig i at konstruksjonen er rar (copy-paste ftl :() - jeg har gjort koden litt kortere basert på dette innspill.

Reply:
defaultValue som "true" er jeg enig er helt tullete, men når jeg skifter til defaultValue "false" og bruker isSimplifiedAndDisabled så går testene hvertfall gjennom. isSimplifiedAndDisabled tar høyde for dette. Er du avhengig å vite at noen har eksplisitt skrudd den av? 
Men men skal ikke krangle om dette, det er småpirk og smak og behag :)

Reply:
Hmm. det betyr at testen ikke er god nok. Dersom jeg har en basefil med 
```yaml
azure:
  azureAppFqdn: "foo.amutv.skead.no"
  groups: []
  jwtToStsConverter:
    enabled: true
```
og jeg har en underfil med
```yaml
azure: false
```
så skal det ikke bli opprettet azure-oppsett, men hvis jeg har en underfil _uten_ azure-oppsett, så skal basefilens innstillinger slå inn.
Default `false` medfører at det siste ikke skjer, så dette mangler jeg åpenbart en test på.

Reply:
Underfil uten azure-oppsett vil jo arve fra basefilen og implisitt få azure: true siden den er "complex"
Resultatet for den app fila blir vel:
```
azure:
  azureAppFqdn: "foo.amutv.skead.no"
  groups: []
  jwtToStsConverter:
    enabled: true
```

Hvis ikke det er caset så må jeg oppdatere min kunnskap om hvordan AuroraConfig arv fungerer

Reply:
Jeg hadde misforstått den funksjonaliteten. Nå fungerer `azure: false` med simplified config.